src/mpc-hc/res/web/variables.html

```diff
@@ -20,6 +20,8 @@
         <p id="durationstring">[durationstring]</p>
         <p id="volumelevel">[volumelevel]</p>
         <p id="muted">[muted]</p>
+        <p id="audiotrack">[audiotrack]</p>

```

Comment:
If might be safer to end the new variables at the end. Just in case people are parsing values in existing order.

Reply:
Fixed.